### PR TITLE
Create bitwarden_rs.sh

### PR DESCRIPTION
**Pull request by szaimen**

### CODE REVIEW

File: apps/talk_signaling.sh [hunk 1]
```diff
@@ -104,7 +104,7 @@ else
 fi
 
 # Let the user choose port. TURN_PORT in msg_box is taken from lib.sh and later changed if user decides to.
-NONO_PORTS=(22 25 53 80 443 3306 5178 5179 5432 7983 8983 10000 8081)
+NONO_PORTS=(22 25 53 80 443 1024 3012 3306 5178 5179 5432 7983 8983 10000 8081)
```
Comment (enoch85) [inline apps/talk_signaling.sh:107]: Great catch! Please add that to the original script as well.

@morph027 Can you please list all the ports in use by the Signaling server?

Comment (szaimen) [reply apps/talk_signaling.sh:107]: they were already added to the original talk script 👍

**Comment by szaimen** [comment]
@enoch85 I've tested this sucessfully. Is anthing else needed to get this in?

**Comment by szaimen** [comment]
The only question that remains from my side is, if we should provide an additional script for bitwarden_rs to disable (or enable) the admin panel completely. But I would to this in a followup-PR, if needed.

**Comment by enoch85** [comment]
This is your script, I won't maintain it ;)

If you think it's good, then I'll make a few adjustments when I have time and then merge. :+1:

**Comment by szaimen** [comment]
> If you think it's good, then I'll make a few adjustments when I have time and then merge. 👍

Yes, I think it is good 👍

**Pull request merged**